config/config_it_test.go

```diff
@@ -566,7 +567,7 @@ func testCgrCfgV1ReloadConfigSection(t *testing.T) {
 			"MaskLen":          0,
 			"NewBranch":        false,
 			"Padding":          "",
-			"RoundingDecimals": 5,
+			"RoundingDecimals": nil,
```

Comment:
Please remove nil from config. The zero values do not need to be specified unless you are overwriting them.